scripts/variantstore/README.md

```diff
@@ -1,9 +1,31 @@
-
-
+# Running the Genome Variant Store (GVS) Pipeline for AoU
+
+## Steps
+1. If this is the first time running the GVS pipeline in a particular Google billing project, create a support ticket for the BigQuery team that includes "enabling cluster metadata pruning support for the BQ Read API."
+2. This list assumes you are starting off with a Terra workspace that contains the re-blocked gVCFs for your callset as "sample sets" in the "Data" tab.  For workflows, you will be following the steps outlined in the [GVS Quickstart workspace](https://app.terra.bio/#workspaces/broad-dsde-firecloud-billing/Genomic%20Variant%20Store%20-%20GVS%20Quickstart).  Make special note of the "Prerequisites" section.
+3. Run the workflows outlined in the Quickstart description in the workspace that contains the data by using the "Copy to Another Workspace" function.  The first two steps (GvsAssignIds and GvsImportGenomes) can be run multiple times to load all necessary sample sets into one "instance" of GVS.  The rest of the steps are only run once, in order, and only after all the samples for the callset are loaded.
+   1. GvsAssignIds at the sample set level ("Step 2")
```

Comment:
```suggestion
   1. GvsAssignIds at the sample set level (Quickstart "Step 2")
```

Reply:
it's not quickstart step 2, it's step 2 in submitting a workflow:
<img width="626" alt="Screen Shot 2021-09-16 at 10 29 10 AM" src="https://user-images.githubusercontent.com/1713505/133630579-83b8d547-6748-4840-8e64-766fbbd0a38a.png">

Reply:
tried to make more clear with 
> "Step 2" in workflow submission